client/src/components/ProgressBar.tsx

```diff
@@ -13,7 +13,9 @@ export default function ProgressBar({ progressBarProps }: Props) {
   const { progressBarRef, audioRef, timeProgress, duration } = progressBarProps;
   function handleProgessChange() {
     if (audioRef.current && progressBarRef.current) {
-      audioRef.current['currentTime'] = progressBarRef.current['value'];
+      audioRef.current!['currentTime'] = Number(
```

Comment:
It would be easier and cleaner to just say `audioRef.current.currentTime` and `progressBarRef.current.value`.

Note that since you check `audioRef.current` and `progressBarRef.current` on line 15, you don't need the `!` here.